### PR TITLE
Adding probation teams health endpoint

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ app.get('/elite2api/health', (req, res) => {
   res.send({ status: 'UP' })
 })
 
+app.get('/probationteams/health/ping', (req, res) => {
+  res.send('pong')
```
Comment (petergphillips) [inline app.js]: shouldn't that be `{ status: 'UP' }` too?

Comment (petergphillips) [reply app.js]: not sure of the url for probation teams but have a look at https://keyworker-api-dev.prison.service.justice.gov.uk/health/ping

**Review by andymarke — APPROVED** [review]
See Peter's comment - I'm guessing you just look at the status and not the JSON

File: app.js (hunk 1 above)
Comment (andrewrlee) [reply app.js]: yes it should be:  https://probation-teams-dev.prison.service.justice.gov.uk/health/ping.
It wont make any diff as it only relies on the status being 200 - but fixed to be consistent.

**Pull request merged**